draft-ietf-rats-corim.md

```diff
@@ -1186,6 +1186,78 @@ Evidence.
 {::include cddl/domain-membership-triple-record.cddl}
 ~~~
 
+
+#### Group Membership Triple {#sec-comid-triple-group-membership}
+
+In order to model hierarchical device composition, CoRIM authors need to identify all components in the composite device (see Composite Device in {{-rats-arch}}). A group describes all target environments that a certain attesting environment is generating evidence about. Consequently, all target environments in question are associated with a common domain via the group-membership-triple.
+
+~~~ cddl
+group-membership-triple-record = [
+  relative-top-layer-root: environment-map
+  relative-sub-nodes: [ + environment-map ]
+]
```

Comment:
I think this doesn't work when different revisions of CoMID have different definitions of what is in an environment

Reply:
> I think this doesn't work when different revisions of CoMID have different definitions of what is in an environment

What doesn't work?

Reply:
I hadn't realised that this is just a rename of domain-membership-triple-record to group-membership-triple-record. I am not convinced that rename makes the document better.

Reply:
tentatively renamed to "environment bundle"